### PR TITLE
Forward protocol: corrections, examples update

### PR DESCRIPTION
**Pull request by spivachuk**

### CODE REVIEW

File: examples/basic.rs [hunk 1]
```diff
@@ -331,9 +579,46 @@ async fn signed_unencrypted() {
...
+        .pack_plaintext(&did_resolver)
+        .await
+        .expect("Unable pack_plaintext");
+
+    // --- Sending message ---
+    println!("Sending message \n{}\n", msg);
+
+    // --- Unpacking message ---
+    let did_resolver = ExampleDIDResolver::new(vec![ALICE_DID_DOC.clone(), BOB_DID_DOC.clone()]);
+    let secrets_resolver = ExampleSecretsResolver::new(BOB_SECRETS.clone());
```
Comment (vimmerru) [inline examples/basic.rs:615]: Maybe keep secrets_resolver empty here to avoid confusion?

**Review by vimmerru — APPROVED** [review]
LGTM, but check my small comment

**Review by vimmerru — DISMISSED** [review]

File: examples/basic.rs (hunk 1 above)
Comment (spivachuk) [reply examples/basic.rs]: Done.

**Review by ashcherbakov — APPROVED** [review]

File: src/protocols/routing/mod.rs [hunk 2]
```diff
@@ -16,6 +16,8 @@ pub use self::forward::ParsedForward;
 
 pub(crate) const FORWARD_MSG_TYPE: &str = "https://didcomm.org/routing/2.0/forward";
 
+pub(crate) const DIDCOMM_V2_PROFILE: &str = "didcomm/v2";
```
Comment (ashcherbakov) [inline src/protocols/routing/mod.rs:21]: We also need to check that Forward protocol version is correct (2.x), see https://github.com/sicpa-dlab/didcomm-python/blob/main/didcomm/core/validators.py#L41.
Let's do it later in the scope of https://github.com/sicpa-dlab/didcomm-gemini/issues/111

**Pull request merged**